app/models/host/managed.rb

```diff
@@ -336,21 +336,24 @@ def handle_realm
     end
   end
 
-  def disk_layout_source
-    @disk_layout_source ||= if disk.present?
-                              Foreman::Renderer::Source::String.new(name: 'Custom disk layout',
-                                                                    content: disk.tr("\r", ''))
-                            elsif ptable.present?
-                              Foreman::Renderer::Source::String.new(name: ptable.name,
-                                                                    content: ptable.layout.tr("\r", ''))
-                            end
+  def disk_layout_source(available_snippets: [])
```

Comment:
As discussed on our internal chat, I'm wondering if the right fix is to use a Database source instead of a string source. At least if the partition table template is actually taken from the database.

Reply:
That would make sense to me :+1: